ngraph/frontend/onnx_import/src/op/upsample.cpp

```diff
@@ -29,180 +29,144 @@ namespace ngraph
         {
             namespace
             {
-                bool check_mode_support(const onnx_import::Node& node, const std::string& mode)
+                constexpr unsigned version_1{1};
```

Comment:
A simple `const` would suffice since those values are not used in any constexpr evaluation later. Just an observation though, not a request to change it.

Reply:
Imo `constexpr` suits better here, as it expresses a nature of the variable (which value will never change) and not an intent of its usage.